README.md

```diff
@@ -1,4 +1,9 @@
 # NGI-NextflowDocs
+
+Project is likely to be out of date and might be incorrect in some case.
```

Comment:
Good to add a massive header to make this more obvious.

Also nice to put all of this text under a blockquote so that it's clearly separate from the rest of the README. eg:

```
> ## Depreciation Notice
> This project has been archived and is no longer maintained. The information contained is likely to be out of date and might be incorrect in some cases.
>
> We recommend...
```

> ## Depreciation Notice
> This project has been archived and is no longer maintained. The information contained is likely to be out of date and might be incorrect in some cases.
>
> We recommend...